lib/constants.js

```diff
@@ -49,22 +49,49 @@ const CoverStatus = {
 
 // gov proposal categories
 const ProposalCategory = {
+  addNewMemberRole: 1,
+  updateMemberRole: 2,
   addCategory: 3,
   editCategory: 4,
   upgradeProxy: 5,
   startEmergencyPause: 6,
-  addEmergencyPause: 7, // extend or switch off emergency pause
+  editEmergencyPause: 7, // extend or switch off emergency pause
+  burnClaims: 8,
+  pauseClaims: 9, // Pause Claim Assessor Voting for 3 days
+  changeCapitalModel: 10,
+  changePricingModel: 11,
+  withdrawFundsForSupportServices: 12,
+  addInvestmentAsset: 13,
+  editInvestmentAssetHolding: 14, // min and max holding percentages
+  editInvestmentAssetStatus: 15,
+  changeABMember: 16,
+  addCurrencyAsset: 17,
+  changeVotingPeriodTo3Days: 18,
+  specialResolution: 19,
+  updateTokenParameters: 20,
+  updateRiskAssessmentParameters: 21,
+  updateGovernanceParameters: 22,
+  updateQuotationParameters: 23,
+  updateClaimsAssessmentParameters: 24,
+  updateInvestmentModuleParameters: 25,
+  updateCapitalModelParameters: 26,
+  changeMasterAddress: 27,
   updateOwnerParameters: 28,
-  upgradeNonProxy: 29,
+  upgradeMultipleContracts: 29,
+  editCurrencyAssetAddress: 30,
+  editCurrencyAssetBaseMin: 31,
+  editInvestmentAssetAddressDecimal: 32,
+  tradingTriggerCheck: 33,
   newContract: 34,
+  editTokenControllerParameters: 35,
   upgradeMCRParameters: 36,
-  upgradeMaster: 37,
+  upgradeMasterAddress: 37,
```

Comment:
Remove the `Address` suffix. This performs a master contract upgrade, the master contract address will not change since it's a proxy.